src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -180,17 +194,49 @@ export const AddressModal: React.FC<Props> = ({
               )}
               <AddressModalFields />
               <Spacer mb={2} />
-              <Input
-                title="Phone number"
-                description="Required for shipping logistics"
-                placeholder="Add phone number"
-                name="phoneNumber"
-                type="tel"
-                onChange={formik.handleChange}
-                onBlur={formik.handleBlur}
-                error={formik.touched.phoneNumber && formik.errors.phoneNumber}
-                value={formik.values?.phoneNumber || ""}
-              />
+              <Flex>
+                <Box style={{ maxWidth: "35%" }}>
+                  <Select
+                    title="Phone number"
+                    description="Only used for shipping purposes"
+                    options={countries}
+                    onSelect={cc => {
+                      setCountryCode(cc)
```

Comment:
Should we be calling `mergeCountryCodeToPhoneNumber` here in case someone changes this value without triggering the `onBlur` in the phone number input?

Reply:
I updated this to account for that edge case...we only call it from here now if there's a value in the phone input field.